gh-codeql

```diff
@@ -81,10 +82,10 @@ if [ "$1" = "set-channel" ]; then
     if [ "$2" != "release" ] && [ "$2" != "nightly" ]; then
         error "Invalid channel: '$2'."
     fi
-    old_channel="$(gh config get extensions.codeql.channel)"
+    old_channel="$(gh config get extensions.codeql.channel 2> /dev/null)" || :
     if [ "${old_channel:-release}" != "$2" ] ; then
-        gh config set extensions.codeql.channel "$2"
-        gh config set extensions.codeql.version ""
+        gh config set extensions.codeql.channel "$2" 2> /dev/null
```

Comment:
Minor, similar to above: Could this be a helper function around `gh config set`? A code comment would also be good in this case, since for `set` we are suppressing a warning, whereas for `get` we are suppressing an error.

Reply:
As above, factoring out into a function is tricky. A comment is definitely sensible, though, I have added one.